site/assets/scss/_component-examples.scss

```diff
@@ -171,6 +171,14 @@
   }
 }
 
+.bd-example-range {
+  .form-range::-moz-range-track,
+  .form-range::-webkit-slider-runnable-track {
```

Comment:
I think you should split those selectors and duplicate styles in order for it to work, shouldn't you? Does this work on both Firefox and Chromium? Isn't the generated CSS splitting them already?